### DISCUSSION TITLE
fcitx5 support?

### DISCUSSION THREAD

**Discussion by malonn23** [tsujan/FeatherPad#785] [Q&A] [answered]
I am just wondering which IME Featherpad supports?  I've read ibus works.  I use `fcitx5` on Arch, and am not able to get the code point to work.  i.e. Ctrl-Shift-U does nothing.  So, before I jump to any conclusions regarding configuration of `fcitx5` on my system, I figured it prudent to ask if Featherpad even supports it.

**Comment by malonn23** [comment, marked as answer]
Indeed, `fcitx5` is supported.  Am I correct to presume that any text editor would support any IME available on any given distro?

**Reply by tsujan** [reply]
FeatherPad supports what Qt6 does.

**Reply by malonn23** [reply]
Yes sir.  I just needed the appropriate dependency, which I didn't have.  Thanks.
